httpql/src/main/java/com/hubspot/httpql/impl/SelectBuilder.java

```diff
@@ -277,7 +277,7 @@ public String getFieldName(String name) {
     select = selectFrom.from(table);
     for (JoinCondition joinCondition : joinConditions) {
       if (joinCondition.isLeftJoin()) {
-        ((SelectJoinStep<?>) select).leftOuterJoin(joinCondition.getTable()).on(joinCondition.getCondition());
+        ((SelectJoinStep<?>) select).leftJoin(joinCondition.getTable()).on(joinCondition.getCondition());
```

Comment:
Just to verify my understanding.. `leftJoin` is equivalent to `leftOuterJoin` right?

Reply:
leftOuterJoin is a convenience method for leftJoin. Seemed odd that we were inconsistent in some places so I cleaned this up.